Tasks/InstallAppleCertificateV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,6 +1,6 @@
 {
   "loc.friendlyName": "Install Apple certificate",
-  "loc.helpMarkDown": "[More information](https://go.microsoft.com/fwlink/?LinkID=862067)",
+  "loc.helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=862067)",
```

Comment:
Is this an intentional change?

Reply:
This is done automatically when we build, its from the localization I guess?